src/app/admin/admin-reports/filtered-collections/filtered-collections.component.html

```diff
@@ -0,0 +1,64 @@
+                          {{ "admin.reports.commons.filters" | translate }}
+                        </ng-template>
+                        <ng-template ngbPanelContent>
+                            <div class="container">
+                                <div class="row">
+                                    <span class="col-3"></span>
+                                    <button class="btn btn-light mt-1 col-6" (click)="submit()">{{ "admin.reports.button.show-collections" | translate }}</button>
+                                </div>
+                                <ds-filters [filtersForm]="filtersFormGroup()"></ds-filters>
+                                <div class="row">
+                                    <span class="col-3"></span>
+                                    <button class="btn btn-light mt-1 col-6" (click)="submit()">{{ "admin.reports.button.show-collections" | translate }}</button>
+                                </div>
+                            </div>
+                        </ng-template>
+                    </ngb-panel>
+                    <ngb-panel id="collections">
+                        <ng-template ngbPanelTitle>
+                          {{ "admin.reports.collections.collections-report" | translate }}
+                        </ng-template>
+                        <ng-template ngbPanelContent>
+                            <table id="table" class="stats">
+                                <thead>
+                                    <tr class="header">
+                                        <th rowspan="2">{{ "admin.reports.collections.community" | translate }}</th>
+                                        <th rowspan="2">{{ "admin.reports.collections.collection" | translate }}</th>
+                                        <th>{{ "admin.reports.collections.nb_items" | translate }}</th>
+                                        <th>{{ "admin.reports.collections.match_all_selected_filters" | translate }}</th>
+                                        <th *ngFor="let filter of results.summary.values | keyvalue">{{ ("admin.reports.commons.filters." + getGroup(filter.key) + "." + filter.key) | translate }}</th>
+                                    </tr>
+                                    <tr class="header">
+                                        <th class="num">{{ results.summary.nbTotalItems }}</th>
+                                        <th class="num">{{ results.summary.allFiltersValue }}</th>
+                                        <th class="num" *ngFor="let filter of results.summary.values | keyvalue">{{ filter.value }}</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    <tr *ngFor="let coll of results.collections">
+                                        <td><a href="{{ coll.communityHandle }}" target="_blank">{{ coll.communityLabel }}</a></td>
```

Comment:
There's a security alert on this line because you are using `target="_blank"` without also specifying this attribute: `rel="noopener noreferrer"`   You should either add that attribute, or remove the target from this link.  For more info, see https://cwe.mitre.org/data/definitions/1022.html